hooks/tk-multi-publish2/maya/collector_integ.py

```diff
@@ -18,13 +20,16 @@ def _resolve_item_fields(self, settings, item):
         # Now run the parent resolve method
         fields = super(MayaFocalLengthCollector, self)._resolve_item_fields(settings, item)
         publisher = self.parent
-        if item.properties.is_sequence:
-            first_frame = int(publisher.util.get_frame_number(item.properties.sequence_paths[0]))
-            end_frame = int(publisher.util.get_frame_number(item.properties.sequence_paths[-1]))
-            # TODO: Update to Dynamic process, When there is a need for different attribute values from the camera.
-            focal_length_dict = self.get_focal_length(first_frame, end_frame)
-            if focal_length_dict:
-                fields.update({"FocalLength": focal_length_dict})
+        all_cameras = cmds.listCameras()
+        if CAMERA_NAME in all_cameras:
```

Comment:
You don't have to get all camera and then filter.
You can do cmds.ls(CAMERA_NAME)

Reply:
Maybe add this:-

target_cam = cmds.ls("CAM")
if not target_cam:
    return fields

Reply:
Thankyou, updated the lines in my code

Reply:
To add on, minor thing, but can we use `focal_length` instead of `FocalLength`? `item.properties` usually uses camelcase (eg. publish_path etc.)...just trying to be consistent.
Also, prefer using `dict[key] = value` over `dict.update({key: val})` as we unnecessarily create an extra dict in the latter method. Also, performance justification: https://stackoverflow.com/questions/15456158/python-dict-update-vs-subscript-to-add-a-single-key-value-pair